src/ui/components/results/paginationcomponent.js

```diff
@@ -164,7 +164,13 @@ export default class PaginationComponent extends Component {
   }
 
   updatePage (offset) {
-    this._onPaginate();
+    const results = this.core.globalStorage.getState(StorageKeys.VERTICAL_RESULTS) || {};
+    let oldOffset = this.core.globalStorage.getState(StorageKeys.SEARCH_OFFSET) || 0;
+    const limit = this.core.globalStorage.getState(StorageKeys.SEARCH_CONFIG).limit;
+    const oldPageNumber = (oldOffset / limit) + 1;
```

Comment:
can limit be 0?

Reply:
Hm, this is probably one of those cases where it could technically happen but would probably not ever happen (since it would mean you get 0 results). I'm somewhat of the opinion that if you set limit to 0 and then attempt to paginate then you deserve whatever implosion you get. They could also attempt to set limit to a negative number and bork themselves that way as well.

@tmeyer2115  do you have thoughts on this?

Reply:
There is validation in the SDK that the limit supplied to a SearchConfig must be between 1 and 50.

Reply:
Minor nit, but currentPageNumber?